packages/amplify-graphql-api-construct-tests/src/__tests__/backends/sql-models/app.ts

```diff
@@ -28,12 +28,21 @@ interface DBDetails {
     };
   };
   dbConnectionConfig: SqlModelDataSourceDbConnectionConfig;
+  schemaConfig: string;
 }
 
+const defaultSchemaConfig = `
```

Comment:
nit (future PR is OK): Give IDE syntax highlighting hints in a comment: 

```suggestion
const defaultSchemaConfig = /* GraphQL */ `
```

Reply:
Comment added for syntax highlighting.